<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClienWidget</class>
 <widget class="QWidget" name="ClienWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>服务器Ip</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditIp">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
